build.ps1

```diff
@@ -23,11 +23,11 @@ if ($env:IntegrationBuildNumber)
         throw $errorMessage
     }
 
-    $buildCommand = { dotnet run --integrationTests }
+    $buildCommand = { dotnet run --integrationTests --skipArtifactGeneration}
 }
 else
 {
-    $buildCommand = { dotnet run --ci }
+    $buildCommand = { dotnet run --ci --skipArtifactGeneration}
```

Comment:
Why are we now skipping the artifact generation for all the build items? (in this file and in build/Program.cs)

Reply:
For the OOP branch we don't want any artifacts generated since the artifact generation/packaging will happen in the third and final pipeline, so I added the param so we wouldn't need to generate it here